diff -urN searchandrescue_1.4.0/sar/config.h searchandrescue_1.4.0-patch/sar/config.h
--- searchandrescue_1.4.0/sar/config.h	2011-04-04 06:12:49.000000000 +1100
+++ searchandrescue_1.4.0-patch/sar/config.h	2011-12-17 04:24:56.440096423 +1100
@@ -24,10 +24,10 @@
 /*
  *      Program version:
  */
-#define PROG_VERSION            "1.3.0"
+#define PROG_VERSION            "1.4.0"
 
 #define PROG_VERSION_MAJOR      1
-#define PROG_VERSION_MINOR      3
+#define PROG_VERSION_MINOR      4
 #define PROG_VERSION_RELEASE	0
 
 
